lib/google/gax/settings.rb

```diff
@@ -400,9 +400,11 @@ def upper_camel_to_lower_underscore(string)
     #   {
     #     "interfaces": {
     #       "google.fake.v1.ServiceName": {
-    #         "retry_codes": {
-    #           "idempotent": ["UNAVAILABLE", "DEADLINE_EXCEEDED"],
-    #           "non_idempotent": []
+    #         "retry_codes_def": {
```

Comment:
`retry_codes_def` is the inner key not the outer key. [Example](https://github.com/googleapis/gax-ruby/blob/master/lib/google/longrunning/operations_client_config.json#L5)